### ISSUE TITLE
 pretransfo (c,K) size mismatch  Error in sedumi 

### ISSUE THREAD

**Issue by jdanielhoyos** [sqlp/sedumi#82]
Hello, thanks for having a look at my problem.

Hello, Im trying with this (using this repo https://github.com/tcunis/sosopt):
pvar mbeta x1 x2;
x = [x1;x2];
V = polydecvar('cv',monomials(x, 1:2))
sosc2 = polyconstr;
sosc2(1) = V-l1>=0; %to make V-l1 SOS I think
sosc2(2) = -((-mbeta-p)s6+V-1)>=0;
sosc2(3) =-((1-V)s8+Vdotfs9+VdotgK*s9+l2)>=0;

            % Solve with gsosopt
            [info2,dopt2,sossol2] = gsosopt(sosc2,x,mbeta,opts)
but it says:
Error using pretransfo
(c,K) size mismatch

          Error in sedumi (line 261)
          [A,b,c,K,prep,origcoeff] = pretransfo(A,b,c,K,pars);
These are the variables:

s6 =
0.078363x1^2 + 0.087278x1x2 + 0.034184x2^2
s8 =
1.5479x1^2 - 0.97065x1x2 + 0.21243x2^2
s9 =
4.1067
Vdotf =
0.16667cv_4x1^4 + 0.33333cv_5x1^3x2 + 0.16667cv_2x1^3 - cv_4
x1^2 - 2cv_5x1x2 - cv_2x1
Vdotg =
2cv_3x1 - cv_4x1 + cv_4x2 - 2cv_5x2 + cv_1 - cv_2
K =
-8.3937x1 + 2.5366x2
l2 =
1.0227e-07x1^4 + 1.0227e-07x2^4
l1 =
1e-06x1^2 + 1e-06x2^2

In other words, only "V" and Vdotf Vdotg have decision variables, the coefficients of V. If you can see the error, I'd appreciate it!

**Comment by mcg1969** [comment]
This is not an appropriate issue for this forum I'm afraid. For one thing it effectively requires the use of sosopt to reproduce. If you can reduce this to an independent demonstration of an issue with SeDuMi, we could consider re-opening this.
